pym/bob/generators/QtCreatorGenerator.py

```diff
@@ -98,7 +104,7 @@ def addBuildConfig(outFile, num, name, buildArgs, buildMeFile):
     outFile.write('<value type="int" key="ProjectExplorer.BuildConfiguration.BuildStepListCount">1</value>\n')
     outFile.write('<value type="bool" key="ProjectExplorer.BuildConfiguration.ClearSystemEnvironment">false</value>\n')
     outFile.write('<valuelist type="QVariantList" key="ProjectExplorer.BuildConfiguration.UserEnvironmentChanges">\n')
-    outFile.write(' <value type="QString">' + "PATH=" + os.environ['PATH'] + '</value>\n')
+    outFile.write(' <value type="QString">' + 'PATH=' + os.environ['PATH'] + '</value>\n')
```

Comment:
Looks like an unneeded change.

Reply:
Correct, I will revert this with next commit.